ui/package.json

```diff
@@ -165,6 +170,8 @@
   },
   "dependencies": {
     "codemirror": "^5.56.0",
+    "curved-arrows": "^0.1.0",
+    "d3": "^7.3.0",
```

Comment:
I would have thought that this would cause a pretty big increase to the file size in vendor.js; but it looks like it's only minimal:

![image](https://user-images.githubusercontent.com/713991/161560419-2262ce3b-7028-4844-a17c-41bd9a4a8bad.png)

Is there perhaps some tree-shaking happening here that I'm not aware of? D3 is a non-trivially large library (~1mb unpacked)

Reply:
Ember has no tree shaking that I know of.

Reply:
probably because we were already pulling in so much from d3 anyway. - but that's a good point that we should be importing the actual package we are interested in instead - `d3-hierarchy` in this case.